src/cpr_data_access/models/search.py

```diff
@@ -35,7 +48,7 @@ class SearchParameters(BaseModel):
     family_ids: Optional[Sequence[str]] = None
     document_ids: Optional[Sequence[str]] = None
 
-    keyword_filters: Optional[Mapping[str, Union[str, Sequence[str]]]] = None
```

Comment:
I like removing this as you've done, allowing str, str and then converting to a list in the keyword_filters_must_be_valid function isn't that clean.

Reply:
the `keyword_filters_must_be_valid` is being removed below, it doesnt add anything, as it repeats the type validation that pydantic does and checks field names, which is done in the backend